Marble Blast Platinum/platinum/client/scripts/taunt.cs

```diff
@@ -121,10 +121,7 @@
 }
 
 function tauntFile(%number) {
-	if ($Audio::CurrentAudioPack $= "")
-		%base = $usermods @ "/data/sound/taunts/";
-	else
-		%base = $usermods @ "/data/sound/ap_" @ $Audio::CurrentAudioPack @ "/";
+	%base = $usermods @ "/data/sound/taunts/";
```

Comment:
Are you sure this is a correct change? I don't see many references to "ap_" in other instances where sound can play, and what about cases where an audio pack doesn't replace the taunt sound? Then there won't be a file at the path location

Reply:
The way taunts play seems really weird to me - it seems those audio profiles get generated on the fly rather than being fixed like most of the rest of the sounds... I'll need to look into it more, cuz I think I see what you're going for here but I don't think it's going to work

Reply:
Yes, before if a sound pack was enabled, taunt sounds would not play, as it was pointing to files which don't exist ("_ap"). However, if there was no audio pack enabled ($Audio::CurrentAudioPack $= ""), the game would play the actual taunt sound (all the taunt sounds are located in "data/sound/taunts"). I think this behaviour was to support custom taunt sounds in sound packs, but no sound packs currently in the game have custom taunt sounds, and as such don't play the taunt sound.

Reply:
oh LOL I was reading it backwards (as in you removed the one line and added in the four lines), and in fact, yeah it explains why I don't hear any taunt sounds normally

We could probably check if the file exists but that seems slightly annoying to do with how the function is currently implemented. Idk if anyone realistically uses custom taunt sounds so this is probably a good change.